src/t8_forest/t8_forest_types.h

```diff
@@ -77,6 +77,8 @@ typedef struct t8_forest
   int                 maxlevel_existing;/**< If >= 0, the maximum occurring refinemnent level of a forest element. */
   int                 do_dup;           /**< Communicator shall be duped. */
   int                 dimension;        /**< Dimension inferred from \b cmesh. */
+  int                 is_incomplete;    /**< Flag to decide whether the forest ist complete or not.
```

Comment:
Please extend the comment here. What does incomplete mean. This is a point where a user possibly sees the concept of a forest with holes for the first time and it should be clear what we mean by an "incomplete" forest. You can also rename the flag here, if you want.